A partial no-op backport of commit 9e8e8bcfb1b1fa359c37499e11a302ec7bde1595
to get caret patches to apply nicely

diff --git a/build/parsePreamble.c b/build/parsePreamble.c
index 5715d2569..7d89617c2 100644
--- a/build/parsePreamble.c
+++ b/build/parsePreamble.c
@@ -22,8 +22,6 @@
 #define SKIPWHITE(_x)	{while (*(_x) && (risspace(*_x) || *(_x) == ',')) (_x)++;}
 #define SKIPNONWHITE(_x){while (*(_x) &&!(risspace(*_x) || *(_x) == ',')) (_x)++;}
 
-#define WHITELIST_NAME ".-_+%{}"
-
 /**
  */
 static const rpmTagVal copyTagsDuringParse[] = {
diff --git a/build/rpmbuild_internal.h b/build/rpmbuild_internal.h
index 439b7d3b5..cc9de88f9 100644
--- a/build/rpmbuild_internal.h
+++ b/build/rpmbuild_internal.h
@@ -17,6 +17,8 @@
 #undef HTKEYTYPE
 #undef HTDATATYPE
 
+#define WHITELIST_NAME ".-_+%{}"
+
 struct TriggerFileEntry {
     int index;
     char * fileName;
